src/app_sensor.c

```diff
@@ -1,21 +1,21 @@
-#include "app_config.h"
```

Comment:
App config is intentionally the first include, that let's app_config override any defaults

Reply:
Fixed in bde68be

Reply:
Easiest way to get the coverage to required level is to add tests for gyroscope data handling. Check how accX, accY, accZ and accXYZ are tested at https://github.com/ruuvi/ruuvi.firmware.c/blob/df0c262db98cbc92b865cd2f5ff5e89b767aa1b9/test/test_app_sensor.c#L761 and repeat the test cases for gyroscope. Definitions are at ruuvi.endpoints.c/ruuvi_endpoints.h